style.css

```diff
@@ -0,0 +1,430 @@
+}
+
+@keyframes header{
+    0%{opacity: 0; left:-30rem;}
+    100%{opacity: 1; left:0;}
+}
+    
+@keyframes personal{
+    0%{opacity: 0; right:-30rem;}
+    100%{opacity: 1; right:0;}
+}
+
+@keyframes skills{
+    0%{opacity: 0; right:-30rem;}
+    100%{opacity: 1; right:0;}
+}
+
+@keyframes education{
+    0%{opacity: 0; right:-30rem;}
+    100%{opacity: 1; right:0;}
+}
+
+
+@keyframes social{
+    0%{opacity: 0; left:-30rem;}
+    100%{opacity: 1; left:0;}
+}
+
+@media (max-width:991px){
+    img{
+        width: 65%;
+        height: auto;
+    }
+    img:hover{
+        width: 75%;
+        max-height: auto;
+    }
+    h2{
+        font-size: 1.6em;
+    }
+    h1{
+        font-size: 2.1em;
+    }
+
+    .content__social{
+        padding: 0 .1rem;
+    }
+    .content__personal{
+        padding: 0 2rem; 
+        line-height: 150%;
+        
+    }
+    .content__personal:hover{
+        padding: .7rem 2rem; 
+        line-height: 150%;
+        word-spacing: 115%;
+        border: 2px solid;
+    }
+}
+
+@media (max-width:758px){
+    img{
+        width: 65%;
+        height: auto;
+    }
+
+    h2{
+        font-size: 1.6rem;
+    }
+    h1{
+        font-size: 2.1rem;
+    }
+
+    .content__social{
+        padding: 0 1rem;
+    }
+
+    .links:hover{
+        border: none;
+        box-shadow: none;
+        font-size: 1em;
+        padding: .6 .5em;
+    }
+    .content__personal{
+        padding: 0 2rem; 
+        line-height: 2rem;
+
+        
+    }
+    .content__personal:hover{
+        padding: .5rem 1rem; 
+        line-height: 2rem;
+        word-spacing: 1rem;
+        border: 2px solid;
+    }
+}
+
+@media (max-width:600px){
+    img{
+        width: 65%;
+        height: auto;
+    }
+    p{
+        font-size: 1.4rem;
+    }
+    h2{
+        font-size: 1.6rem;
+    }
+    h1{
+        font-size: 2rem;
+    }
+
+    .header header-picture{
+        padding-top: 1rem;
+    }
+
+   .container{
+    display: flex;
+    flex-flow:column;
+   }
+
+    .content__skills{
+        padding: 0 1em;
+    }
+
+    .content__header{
+        border-radius: 20% 20% 0 0;
+    }
+
+    .content__personal{
+        padding-right: 1rem; 
+        border-radius: 0;
+    }
+
+    .content__education{
+        padding: .5em;
+        border-radius: 0 0 20% 20%;
+    }
+
+    .content__social{
+        border-radius: 0;
+    }
+
+    .links{
+        font-size: .7rem;
+    }
+
+    .links:hover{
+        font-size: .7rem;
+        padding: 6 .5em;
+    }
+
+    .content__education:hover{
+        padding: 0;
+        border: 2px solid black;
+    }
+
+    .content__personal:hover{
+        padding: 0 2.1rem; 
+        border: 2px solid black;
+    }
+    .content__skills:hover{
+        border: 2px solid black;
+    }
+    .content__social:hover{
+        border: 2px solid black;
+    }
+    .content__header:hover{
+        border: 2px solid black;
+    }
+    .mailtxt:hover {
+        font-size: .75em;
+    }
+}
+
```

Comment:
A common file ending is one empty line, you could remove the extra lines.

Reply:
Fixed.